langchain/src/vectorstores/pinecone.ts

```diff
@@ -89,13 +89,16 @@ export class PineconeStore extends VectorStore {
   }
 
   static async fromTexts(
-    pineconeClient: VectorOperationsApi,
     texts: string[],
     metadatas: object[],
     embeddings: Embeddings,
-    textKey = "text",
-    namespace: string | undefined = undefined
+    dbConfig: {
+      pineconeClient: VectorOperationsApi;
+      textKey: string;
+      namespace: string | undefined;
+    }
```

Comment:
i had to change the type signature of the `fromTexts` methods of these vector stores in order to standardize them enough for the base class, in order to use `fromTexts` in example selectors. idk if other docs need to be updated, or if they need to be notified